internal/pkg/manifest/svc.go

```diff
@@ -88,25 +149,49 @@ func (c *Count) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
-// Autoscaling represents the configurable options for Auto Scaling as well as
+// Desired returns the desiredCount to be set on the CFN template
+func (c *Count) Desired() (*int, error) {
+	desiredCount := c.Value
```

Comment:
nit: can we flip the conditionals to use guard-statements to reduce nesting?
```go
if c.AdvancedCount.IsEmpty() {
   return c.Value
}
if c.AdvancedCount.IgnoreRange() {
  return c.AdvancedCount.Spot
}
...
```

Reply:
https://github.com/aws/copilot-cli/pull/2188/commits/f6362c96507dd91f2b2bed70c8b93ae26f2cba73